doc/nrf/releases_and_maturity/migration/migration_3.1_54h_suit_ironside.rst

```diff
@@ -0,0 +1,177 @@
+.. _migration_3.1_54h_suit_ironside:
+
+Migration from nRF Connect SDK 3.0 (SUIT) to 3.1 (IronSide SE) on nRF54H20 SoC
+==============================================================================
+
+.. contents::
+   :local:
+   :depth: 2
+
+This document explains how to migrate your existing |NCS| v3.0 application for the nRF54H20 SoC running SUIT to the |NCS| v3.1 using IronSide SE.
+
+To follow this guide, you must meet the following prerequisites:
+
+* You have a working |NCS| 3.0 application for the nRF54H20 SoC using SUIT.
+* You have installed |NCS| 3.1 and its toolchain.
+
+.. note::
+   To program IronSide SE on your nRF54H20 SoC-based device, your device must be in lifecycle state (LCS) ``EMPTY``.
+   Devices using SUIT in LCS RoT cannot be transitioned back to LCS EMPTY.
+
+Breaking Changes
+================
+
+The following is a summary of the breaking changes that apply when migrating applications:
+
+* SUIT support is removed in |NCS| 3.1.
+  The Secure Domain now runs IronSide SE, which is agnostic to the device firmware update (DFU) mechanism used.
+  DFU is handled by the local domain (for example, using MCUboot).
+* IronSide SE uses a new UICR format and moves the peripheral configuration into a dedicated MRAM partition.
+* The shared memory used for communication with the Secure Domain is now located in a fixed RAM-20 region.
+  You no longer reserve it manually.
+* DFU support moves from SUIT to MCUboot.
+  You must enable and configure MCUboot in your project.
+
+Update prj.conf
+---------------
+
+Remove the following SUIT-specific Kconfig options:
+
+* :kconfig:option:`CONFIG_SUIT`
+* :kconfig:option:`CONFIG_SUIT_SECURE_DOMAIN`
+
+Enable MCUboot for device firmware update (DFU) in the :file:`sysbuild.conf` file as follows:
+
+* Set :kconfig:option:`SB_CONFIG_BOOTLOADER_MCUBOOT` to ``y``.
```

Comment:
Are these steps to follow one after another? Should this maybe be a step list or an unordered list?

Reply:
These are kconfig options that need to be set (or removed). It does not matter the order in which they are set, they are all applied at build time. However, we could order them just to highlight a logical order to the actions prescribed.